packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -158,6 +158,10 @@ function ChangePassword({
 
   const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
+    if (!user) {
+      return;
+    }
```

Comment:
Technically, `user` will always be defined because `ChangePassword` does not render until there's a valid user.

But TS doesn't know this because `handleSubmit` is a callback, so I'm adding this additional check here.